raiden_contracts/deploy/__main__.py

```diff
@@ -97,74 +110,86 @@ def main(**kwargs):
     print('Owner is', owner)
     assert web3.eth.getBalance(owner) > 0, 'Account with insuficient funds.'
 
-    transaction = {'from': owner}
-    if gas_price == 0:
-        transaction['gasPrice'] = gas_price
+    password = getpass(f'Enter password for {owner}:')
```

Comment:
I assume this works well with a node that's running with an unlocked account - meaning, it doesn't prompt you for the password.

Reply:
Hm, good question, I'll test it later.